catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -461,8 +473,22 @@ protected void renameItem(T item, String newName) {
 		finishActionMode();
 	}
 
-	protected void mergeProjects(List<T> selectedProjects, String mergeProjectName) {
-		ToastUtil.showSuccess(getContext(), R.string.merging_project_text);
+	protected void mergeProjects(List<T> selectedProjects, String mergeProjectName) throws IOException, LoadingProjectException {
```

Comment:
merging projects is only available for exactly 2 projects, so pick those two out wherever you call this and refactor this to accept exactly two input projects

Reply:
also all of this could be moved out to the "MergeProject" class, to at least have everything merge related in one place.
and as stated below, im not happy with the name "MergeProject"